pages/css/documentation.md

```diff
@@ -4,15 +4,33 @@ permalink: /css/documentation/
 sidenav: css
 ---
 # Documentation
+
+You should always document:
+- Magic numbers (when you absolutely cannot avoid using them)
+- Overrides of imported frameworks like USWDS
+- Rules that must be used in conjunction with one another
+(using BEM naming conventions is somewhat self-documenting in this regard)
+
+We also recommend maintaining a project style guide of your choice.
+Add examples of how to use classes that make up complex modules
+when used in conjunction with one another.
+
 ## Sass Comments
 Be intentional when you use `//` (silent comments) versus `/* */`
 (which are preserved in the CSS output). When in doubt, use `//`.
 
 ## KSS
-Use KSS for documentation. More information on KSS can be found on the
-[official site](http://warpspire.com/kss/).
+KSS is the most common CSS documentation method to date. It relies
+on comments that follow a specific format to generate documentation
+for you. The generated documentation can be modified through templates.
+More information on KSS can be found on the
+[official KSS site](http://warpspire.com/kss/).
 
-### Example
+The original KSS repo is not maintained. There are numerous forks that are
```

Comment:
Is it accurate to say that it is not maintained? Is it not updated because it's abandoned, or because it's mature and needs little to no updating?

Reply:
How about something like "While the original KSS repo has not been updated in several years, community development has continued in forks and it remains one of the most used CSS documentation tools"